### ISSUE TITLE
Optimize installation of Dynatrace Monitoring Support to include SLI service

### ISSUE THREAD

**Issue by grabnerandi** [keptn-contrib/dynatrace-service#117]
Right now when enabling support for Dynatrace a user has to install the DYnatrace Service as well as the DYnatrace SLI service. installing the Dynatrace Service is done via keptn configure monitoring dynatrace - yet - there is an additional step to install & configure the Dynatrace SLI service

From an end user perspective this is an error prone task. We have seen some users not installing the SLI PRovider as they assumed they already enabled Dynatrace support.

I therefore suggest to optimize the installation of Dynatrace Monitoring support so that all necessary components are automatically installed and configured

**Comment by grabnerandi** [comment]
I want to propose a couple of options how this can be made easier for the end user

```
keptn install capability=dynatrace --tenent=mytenent --api-token=ABCD --paas-token=FERGT
```

This command should automatically do these things
1. validate that the tenant URL can be reached and that the tokens work, e.g: execute a couple of REST Calls that valide the token has the correct priviliges
2. create the dynatrace secret
3. install dynatrace-service
4. install dynatrace-sli service
5. print out instructions what was installed and provide instructions on how to uninstall these components again, e.g: keptn uninstall capability=dynatrace

**Comment by arthurpitman** [comment]
Closing this as it will be addressed as part of #283
